### PR TITLE
Add CLI output alongside lint rule example docs - fixes #822

### LINKED ISSUE #822: Add CLI output alongside lint rule example docs
We can automatically generate this.

### PR DESCRIPTION
**Pull request by sebmck**
Layout isn't the best, but the hard implementation part is done now.

### CODE REVIEW

File: website/src/docs/lint/rules/js/caseSingleStatement.md [hunk 1]
```diff
@@ -17,32 +17,49 @@ MISSING DOCUMENTATION
...
-	}
-}
-```
-```typescript
-switch (foo) {
-	case true:
-}
-```
+### Invalid
+<pre class="language-text"><code class="language-text"><span class="token keyword">switch</span> <span class="token punctuation">(</span><span class="token variable">foo</span><span class="token punctuation">)</span> <span class="token punctuation">{</span>
```
Comment (ematipico) [inline website/src/docs/lint/rules/js/caseSingleStatement.md:21]: Is this expected?

Comment (sebmck) [reply website/src/docs/lint/rules/js/caseSingleStatement.md:21]: Sorry, is what expected?

Comment (ematipico) [reply website/src/docs/lint/rules/js/caseSingleStatement.md:21]: The snapshot test has now printed HTML instead of normal text

Comment (sebmck) [reply website/src/docs/lint/rules/js/caseSingleStatement.md:21]: Yeah it's the code ran through our syntax highlighter which is more rich than what PrismJS provides. We can do it here so why not. We still emit PrismJS CSS classes though.

**Comment by sebmck** [comment]
Example: https://deploy-preview-825--romefrontend.netlify.app/docs/lint/rules/js/noemptycharacterclass/#js%2Fnoemptycharacterclass

Screen Shot 2020-07-17 at 12 13 01 AM

**Comment by ematipico** [comment]
I get it know! It's a really nice improvement!

**Pull request merged**